arkouda/io.py

```diff
@@ -1572,7 +1572,8 @@ def load(
     with a .parquet appended to the prefix_path.
     Parquet files were previously ALWAYS stored with a ``.parquet`` extension.
 
-    This function will be deprecated when glob flags are added to read_* functions
+    ak.load does not support loading a single file.
+    For loading single HDF5 files without the _LOCALE#### suffix please use ak.read().
```

Comment:
Should this be "use ak.read_hdf()"?

Reply:
No, `ak.load` can read HDF5, parquet, and CSV. Thus, we want this to be pointed to the generic `read` as well. HDF5 is just the one currently supporting writing to a single file, but all should support reading single files.

Reply:
Makes sense. Thank you for clarifying.